components/shared/code/libs/lib_nvm.c

```diff
@@ -227,30 +242,65 @@ void lib_nvm_init(void)
 
     cycleLog.totalCycles++;
     lib_nvm_requestWrite(NVM_ENTRYID_CYCLE);
+
+#if FEATURE_IS_ENABLED(NVM_TASK)
+    data.queue_handle = xQueueCreateStatic(NVM_ENTRYID_COUNT,
+                                           sizeof(lib_nvm_entryAction_S),
+                                           data.entry_action_queue_buff,
+                                           &data.entry_action_queue);
+#endif
+}
+
+void lib_nvm_start(void)
+{
+#if FEATURE_IS_ENABLED(NVM_TASK)
+    data.rtosStarted = true;
```

Comment:
I don't understand why this is necessary. None of these tasks will run until the rtos is started, right?

Reply:
So is this actually just an indication that NVM setup is done? If so, maybe update the name to reflect that and make it accessible from other modules either with a global or a accessor.

`nvmInitComplete` or something